app/helpers/menu_helper/common_menu.rb

```diff
@@ -9,11 +9,20 @@ def initialize(user, context)
     end
 
     def items
-      [
-        item(t('menu.bill_of_materials'), context.bill_of_materials_path),
-        item(t('menu.items'), context.items_path),
-        item(t('menu.unit_of_measures'), context.unit_of_measures_path)
-      ]
+      menu_options.each_with_object([]) do |(key, value), results|
+        results << item(t("menu.#{key}"), context.method(value).call)
+      end
+    end
+
+    private
+
+    def menu_options
```

Comment:
I feel like you may be able to boil this down to an array of things, remapped with `_path`... for example:

```ruby
%w[bill_of_materials customers].map { |prefix| "#{prefix}_path" }
```